delete.go

```diff
@@ -39,7 +42,41 @@ EXAMPLE:
 		},
 	},
 	Action: func(context *cli.Context) error {
-		// TODO
-		return nil
+		return delete(context.String("container-id"), context.Bool("force"))
 	},
 }
+
+func delete(containerID string, force bool) error {
+	// Checks the MUST and MUST NOT from OCI runtime specification
+	if err := checkDeleteParamsIntegrity(containerID); err != nil {
+		return err
+	}
+
+	if _, err := vc.StopPod(containerID); err != nil {
+		return err
+	}
+
```

Comment:
I guess we can do the StopPod only if we force. And do only the Delete in case force == false.